test/utf8_test.dart

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) 2019, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:test/test.dart';
+import 'package:ffi/ffi.dart';
+
+main() {
+  test("fromUtf8 . toUtf8 is identity", () {
+    final String start = "Hello World!\n";
+    final String end = Utf8.fromUtf8(Utf8.toUtf8(start));
+    expect(end, equals(start));
+  });
```

Comment:
This does not actually test that it encodes or decodes correctly (it could be storing UTF-16 in the middle).

Consider testing each function independently, and test some edge cases, like invalid UTF-8 or Dart strings containing unpaired surrogates or NUL characters, or an input of more than 4096 bytes.

Reply:
Done.